_api_app/app/Http/Controllers/SettingsController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\SiteSettings;
+use Illuminate\Http\Request;
+
+class SettingsController extends Controller
+{
+
+    public function update(Request $request) {
+        $json = $request->json()->all();
+        $path_arr = explode('/', $json['path']);
+        $site = $path_arr[0];
+        $settings = new SiteSettings($site);
+        $path_arr = array_slice($path_arr, 1);
+
+        $res = $settings->saveValueByPath($json['path'], $json['value']);
+        // @@@:TODO: Replace this with something sensible, when migration to redux is done
```

Comment:
how is this meant? What the fields `update` and `real` are supposed to contain?

Reply:
`update` is an updated value, `real` as a  transformed and formatted value for old Berta front end needs (url prefixes, date format etc).

Reply:
So the idea is that the `update` field will contain some kind of new value when the migration to redux is done, but the `real` field will contain backward compatible value?